### ISSUE TITLE
test for mobile

### ISSUE THREAD

**Issue by ericnost** [ericnost/streamlit_test#82]
FWIW, it looks pretty good across devices. The only hang up I see is the maps and colorscales, which have fixed widths, sometimes bleed over. Seems ok on tablets though compared to phones.

**Comment by Frijol** [comment]
Yeah I think it is basically good for mobile now! I asked about the need for mobile compatibility mostly because I'd like to add features that might not work well on mobile, e.g. side by side map comparisons
